app.vue

```diff
@@ -8,12 +8,12 @@
               <v-stage :config="configStage">
                 <v-layer>
                   <Grid :spacing="50" />
-                  <Point :x="10" :y="20" />
-                  <ForceVector v-for="vector in forceVectors" 
-                    :key="vector.id"
+                  <ForceVector v-for="vector in forceVectors" :key="vectorIndex"
```

Comment:
I was reading that it isn't recommended to use the index as a key, since if the order of the items changes, then vue might not rerender correctly. I think this should work if we use `vector.id.`